src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -180,9 +199,47 @@
             :content-fingerprints="nftISCNContentFingerprints"
           />
         </section>
+
+        <!-- phone: footer -->
+        <section class="flex-col items-center hidden phone:flex">
+          <ButtonV2
+            class="text-medium-gray"
+            content-class="text-[12px]"
+            preset="plain"
+            size="mini"
+            :to="{ name: 'nft-class-classId', params: { classId } }"
+          >
+            <template #prepend>
+              <IconEye class="w-[12px] h-[12px]" />
+            </template>
+            {{ $t('nft_details_page_button_view_details') }}
+          </ButtonV2>
+          <hr
+            :class="[
+              'hidden',
+              'phone:block',
+              'h-[2px]',
+              'w-full',
+              'my-[24px]',
+              'border-shade-gray',
+            ]"
+          >
+          <NFTPageControlBar
+            :collected-count="ownCount"
+            :collected-nft-ids="userCollectedNFTList"
+            :class-id="classId"
+            :current-nft-id="nftId"
+            :view="nftIsNew ? 'created' : 'collected'"
+            :price="NFTPrice"
+            :is-writing-nft="nftIsWritingNFT"
+            @transfer="onToggleTransfer"
+            @collect="handleCollectFromControlBar"
+            @click-user-collected-count="handleClickUserCollectedCount"
+          />
+        </section>
       </div>
     </div>
-
+    <NuxtChild v-else-if="!isLoading && isMessagePage"/>
```

Comment:
I don't think you need to`isMessagePage` to the condition as this is induced by route
If no nested route, nothing is rendered

Reply:
I still hope to retain 'isMessagePage' for better readability. Therefore, I added keep-alive to improve the white screen problem when changing pages. what do you think?

https://user-images.githubusercontent.com/75730405/208253404-6547fc97-b07e-471f-a5df-6d3996afe945.mov